modules/overriding-default-node-ip-selection-logic.adoc

```diff
@@ -33,7 +33,7 @@ NODEIP_HINT=192.0.2.1
 +
 [source,terminal]
 ----
-$ echo 'NODEIP_HINT=192.0.2.1' | base64
+$ echo -n 'NODEIP_HINT=192.0.2.1' | base64
```

Comment:
`base64 -w0` to disable line wrapping

Reply:
@rbbratta fixed. PTAL when you get a moment. 

 Thanks a lot!